### ISSUE TITLE
What about LIDAR and IMU?

### ISSUE THREAD

**Issue by Bercon** [castacks/tartanair_tools#5]
The paper and website for this dataset talks about "simulated multi-line LiDAR points, and simulated IMU readings.". However, this repo doesn't mention anything about those. Are they going to be made public and added later?

**Comment by Amigoshan** [comment]
Right! For now it's not available yet. We might be able to add IMU data soon. But for the LiDAR data, we are still figuring that out. One biggest issue is the balance between the data size and LiDAR resolution. To simulate a Velodyne 16-line need a huge amount of storage space. It's not practical for us to publish such large data, and not practical for users to download. 
Could you talk about what's your use case with the LiDAR data? Maybe we can figure out a practical way to do this.

**Comment by Bercon** [comment]
Thanks for a quick reply! Actually for my use case I'd be more interested in the IMU data. Great to hear that it might be available soon!

I asked about both, because I couldn't find them and was wonder if I missed something. However, I don't have immediate need for the LiDAR data.

**Comment by Hezip** [comment]
In some cases, I want to test both visual slam and lidar slam. I do like to have a lidar dataset~. Maybe you can share the tool to generate lidar points from depth images. It can save storage space. By the way, I can also generate the occupant grid map from lidar points...
Looking forward to your reply and thank you very much!

**Comment by betaBison** [comment]
+1 I'm also looking forward to IMU data being available if possible. I've enjoyed working with what's already here so far!

**Comment by karnikram** [comment]
+1 for IMU data! Is it still in the works?

**Comment by ZuoJiaxing** [comment]
+1 for IMU data .......

**Comment by brandonwagstaff** [comment]
+1 for IMU data! Hopefully this is still coming

**Comment by alsarmie** [comment]
+1 for IMU data being available.

**Comment by arenas7307979** [comment]
+1 I'm also looking forward to IMU data being available if possible.

**Comment by WangWen-Believer** [comment]
+1 for IMU data

**Comment by tianxiaguixin002** [comment]
+1 for IMU data and lidar scan data

**Comment by LongruiDong** [comment]
+1 for ground truth occupancy map

**Comment by fengshiyu58** [comment]
waiting for IMU data +1

**Comment by tetaud-sebastien** [comment]
The LiDAR data has been released?

**Comment by shneka-swamy** [comment]
Is the IMU data released?

**Comment by wenhuiwei-ustc** [comment]
waiting for IMU data

**Comment by Neoyning** [comment]
Hey, I'm see there are the Multimodal ground truth labels on the web page, looking forward to the update~

**Comment by sebastien-tetaud** [comment]
You can forget guys :) I have been waiting since a year 👎🏼
